lib/templates.rake

```diff
@@ -103,6 +107,23 @@ def update_snippet
   { :diff => diff, :status => status, :result => result }
 end
 
+def get_default_branch
+  # Always use develop on Foreman-nightly
+  return "-b develop" if Foreman::Version.new.tag == "develop"
+
+  url = URI.parse("https://api.github.com/repos/theforeman/community-templates/branches")
+  req = Net::HTTP.new(url.host, url.port)
+  req.use_ssl = true
+  res = req.get(url.path)
+  repodata = JSON.parse(res.body)
+  version = Foreman::Version.new.short
```

Comment:
`SETTINGS[:version]` is a Foreman::Version object, so `SETTINGS[:version].tag` or .short, notag etc will work.